### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Inappropriate control type for the edit in "Container Registry-Open ACR Explorer dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1412]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Push Image.
4. In "push Image" dialog ,fill all the required information and select ok button.
5. Navigate to and select "Show Azure Explorer" button at the toolbar .
6. In "Azure Explorer" , select the pushed image under container registries and use shift+F10 to open the context menu.
7. Navigate to and select "Open ACR Explorer" menu item .
8. Tab to "register name , type , subscription id , region, resource group and login server URL" and observe the control type using inspect.

**Actual Result:**
Control type is document

**Narrator :**
Tab
Type, 
Tab
User Name, 
Tab
Resource Group, 
Tab
Copy to Clipboard, link,
Subscription Id, 
Copy to Clipboard, link,
Region, 
Login Server URL,  

**NVDA:**
Registry Name  edit  multi line  read only
monaacr2
Disable  button
Type  edit  multi line  read only
Microsoft.ContainerRegistry/registries
User Name  edit  multi line  read only
monaacr2
Resource Group  edit  multi line  read only
monatest
Copy to Clipboard  link
Copy to Clipboard  link
Subscription Id  edit  multi line  read only
ffa52f27-be12-4cad-b1ea-c2c241b6cceb
Copy to Clipboard  link
Copy to Clipboard  link
Region  edit  multi line  read only
eastus
Login Server URL  edit  multi line  read only
monaacr2.azurecr.io.

**Expected Result:**
Control type should be Edit.

Narrator should read as
Type ,  read only
Microsoft.ContainerRegistry/registries
User Name  read only
monaacr2
Resource Group  read only
monatest
Subscription Id read only
ffa52f27-be12-4cad-b1ea-c2c241b6cceb
Region  read only
eastus
Login Server URL  read only
monaacr2.azurecr.io

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![acr explorer-narrator](https://user-images.githubusercontent.com/22956130/35686127-900960a6-0791-11e8-8b82-2ce3fe2adf3c.jpg)
![acr explorer-nvda](https://user-images.githubusercontent.com/22956130/35686134-92a46c8e-0791-11e8-9103-8e4d56a19acc.jpg)

**Comment by Eskibear** [comment]
under investigation

**Comment by jdneo** [comment]
After fix:
![wechat screenshot_20180412104957](https://user-images.githubusercontent.com/6193897/38654122-5eb41a06-3e40-11e8-9f87-369ce3fe2be5.png)

This aligns with the Eclipse's official field:
![wechat screenshot_20180412103715](https://user-images.githubusercontent.com/6193897/38654148-7940ecd2-3e40-11e8-88c7-2f2a6d96b659.png)

**Comment by WiproPSAccTest** [comment]
Verified the issue in V.3.6.0 .Issue got fixed.
Please add closed tag.
![1412-fixed](https://user-images.githubusercontent.com/22956130/39760460-f74cf046-52f2-11e8-9ee8-1318c1484d0d.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749875
